File: LeetCode-Programs/Height Checker.py
class Solution:
    def heightChecker(self, heights: List[int]) -> int:
        new_list=heights.copy()
        heights.sort()
        count=0
        for i in range(len(heights)):
            if new_list[i]!=heights[i]:
                count=count+1
        return count

File: LeetCode-Programs/Height Checker.cpp
class Solution {
public:
    int heightChecker(vector<int>& heights) {
        vector <int>vec=heights;
        sort(heights.begin(),heights.end());
        int count=0;
        for(int i=0;i<heights.size();i++)
        {
            if(heights[i]!=vec[i])
            {
                count++;
            }
        } 
        return count;
    }
};
